torch/_inductor/scheduler.py

```diff
@@ -1234,6 +1317,8 @@ def codegen(self):
                 self.get_backend(device).codegen_template(node, epilogue)
             elif node.is_extern():
                 self.codegen_extern_call(node)
+            elif node.is_foreach():
+                self.get_backend(device).codegen_foreach(node)
```

Comment:
Make sure the C++ backend throws a good error message for this.  Or maybe just don't create ForEach nodes on C++

Reply:
Yeah I'll fallback for c++